Classes/Integrations/DiscordClient.cs

```diff
@@ -31,6 +31,19 @@ public class DiscordClient {
 
 		client.SetPresence(rp);
 	}
+
+	private string GetPresenceDetails(string songName)
+	{
+		switch (songName)
+		{
+			case null: return "No song open";
+			case "": return "Mapping: *Untitled*";
```

Comment:
Do the asterisks actually italicise the text in Discord? If not we should pick a different phrasing like "Working on an untitled map" or something like that.

Reply:
They don't. I'll update the phrasing here